src/argilla/server/models.py

```diff
@@ -56,7 +56,9 @@ class Workspace(Base):
     inserted_at: Mapped[datetime] = mapped_column(default=datetime.utcnow)
     updated_at: Mapped[datetime] = mapped_column(default=default_inserted_at, onupdate=datetime.utcnow)
 
-    users: Mapped[List["User"]] = relationship(secondary="users_workspaces", back_populates="workspaces")
+    users: Mapped[List["User"]] = relationship(
+        secondary="users_workspaces", back_populates="workspaces", order_by="asc(UserWorkspace.inserted_at)"
```

Comment:
I have added here a default order for `users` and the same one later for `workspaces`.

Reply:
Could we use the model field instead of evaluating a string? I don't like using strings refs since names refactoring can be hell.

```python
from sqlalchemy import asc

...
order_by=asc(UserWorkspace.inserted_at)
...

```